public_html/admin/controller/pages/sale/contact.php

```diff
@@ -27,8 +28,7 @@
 
 class ControllerPagesSaleContact extends AController
 {
-    public $data = array();
```

Comment:
$data is use below. Why it is removed?

Reply:
this is an inherited property from the parent class